### PR TITLE
The mobile and the desktop version

### PR DESCRIPTION
**Pull request by Marcraphael12**
In this PR we are checking if:

- The webpage is mobile (365px - 768px)
- The webpage is desktop (768px - wider)
- The page is responsive
- We applied HTML&CSS best practices
- The mobile menu is showing and hiding
- The page follows the project requirements (personalize the content (!important), but follow the colors, typography (font face,    size, and weight), and, the layout (composition and space between elements)).
[demo video](https://www.loom.com/share/6b10ca9b295148cbb004fde9262ba33e?sharedAppSource=personal_library)

Feel free to check my work ad if there is an issue, please, kindly leave a comment or visit the issue page. 
Thanks for your support!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+						<h4 class="post">
+							Nemo quaeso miretur, si post exsudatos labores itinerum longos
+						</h4>
+						<p class="testimony">
+							Nemo quaeso miretur, si post exsudatos labores itinerum longos congestosque adfatim commeatus fiducia vestri ductante barbaricos pagos adventans.
+						</p>
+					</div>
+				</li>
+			</ul>
+		</section> -->
```
Comment (carlos-ssh) [inline index.html]: - Remove unused code. Keep your project clean.
- Add a Link of Video presentation
_Provide feedback about the video presentation in the code review summary._

- The recorded video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- The video needs to have a demo of the project, showing the pages in different screen sizes.
- The video needs to have a highlight about a piece of the code the student finds interesting or is proud of making.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+		<!-- <section class="partners column-center">
+			<h2 class="column-center">Partner</h2>
+			<ul class="partner-list inline-center">
+				<li><img src="./images/photos/behance.svg" alt="behance link" title="behance" class="partner"></li>
+				<li><img src="./images/photos/github.svg" alt="github link" title="github" class="partner"></li>
+        <li><img src="./images/photos/microverse.jpeg" alt="microverse link" title="microverse" class="partner"></li>
+				<li><img src="./images/photos/google.svg" alt="Google link" title="google" class="partner"></li>
+				<li><img src="./images/photos/atom.png" alt="atom link" title="atom code editor" class="partner"></li>
+			</ul>
+		</section> -->
```
Comment (carlos-ssh) [inline about.html:124]: - Remove unused code. Keep your project clean.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,576 @@
...
+  list-style: none;
+  text-decoration: none;
+  color: var(--white);
+  font-family: 'Lato', sans-serif;
+}
+
+body { background-color: var(--gray); }
+
+.main {
+  background-image: url(./images/backgrounds/students.jpg);
```
Comment (carlos-ssh) [inline style.css:24]: - Fix the size of your image on desktop view:
Screen Shot 2021-09-13 at 18 50 23

File: index.html [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+  <!-- JavaScipt links -->
+
+  <title>Document</title>
+</head>
+
+<body>
+  <header class="page-header column-center">
+    <button type="button" name="button" class="menu">
+      <img src="./images/buttons/hamburger.svg" alt="menu">
```
Comment (carlos-ssh) [inline index.html:40]: - Your hamburger button isn't displaying the menu. Kindly work on it.
- Fix the button position.
Screen Shot 2021-09-13 at 19 01 40

File: style.css [hunk 5]
```diff
@@ -0,0 +1,576 @@
+/* Common style: This part is for both home and about page */
+:root {
+  --dark-text: #272a31;
+  --dark-background: #313235;
+  --dark-gray: #797c85;
+  --red: #fa4229;
+  --gray: #f7f7f7;
+  --white: white;
+
+  /* calling: var(--color) */
```
Comment (carlos-ssh) [inline style.css]: - Remove unused lines of code.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Marcraphael12 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Marcraphael12) [reply index.html]: done

File: about.html (hunk 2 above)
Comment (Marcraphael12) [reply about.html:124]: done

File: style.css (hunk 3 above)
Comment (Marcraphael12) [reply style.css:24]: done

File: style.css (hunk 5 above)
Comment (Marcraphael12) [reply style.css]: done

Comment (Marcraphael12) [reply style.css]: done

**Review by dasileker — APPROVED** [review]
### **Hi @Marcraphael12,** 

### **Project Approved ✅**

Approved

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**